### PR TITLE
[Improvement][Metrics] Filtering data by application in grafana-demo worker dashboard example

### PR DESCRIPTION
**Pull request by xuchunlai**
## Purpose of the pull request
#9324
Add filter condition to improve performance

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Comment by DarkAssassinator** [comment]
hi @xuchunlai , could u please update the title?

**Comment by xuchunlai** [comment]
> hi @xuchunlai , could u please update the title?

done

**Review by EricGao888 — DISMISSED** [review]

**Comment by EricGao888** [comment]
@xuchunlai If you want to add a filtering for `application`, why do u only add it for `process_cpu_usage`?

**Comment by xuchunlai** [comment]
> @xuchunlai If you want to add a filtering for `application`, why do u only add it for `process_cpu_usage`?

This metirc is the only one that has no filter condition in worker-server, and is slow to load when used.

**Comment by xuchunlai** [comment]
> @xuchunlai If you want to add a filtering for `application`, why do u only add it for `process_cpu_usage`?

Many systems contain the metric 'process_start_time_seconds', and load it very slow when used. use filter condition 'application' to query the metric of DS only.

File: dolphinscheduler-api/src/main/resources/swagger.properties [hunk 1]
```diff
@@ -15,4 +15,5 @@
 # limitations under the License.
 #
 
+#test
```
Comment (kezhenxu94) [inline dolphinscheduler-api/src/main/resources/swagger.properties]: Remove this

```suggestion
```

**Review by EricGao888 — APPROVED** [review]
Waiting for CI to pass.

**Pull request merged**